### PR TITLE
Done

### PR DESCRIPTION
**Pull request by vovkandrew**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2) ? 2 :
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
...
+                (((result1 < result2) && (bet1 < bet2))
+                        || ((result1 > result2) && (bet1 > bet2))
+                        || ((result1 == result2) && (bet1 == bet2))) ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: same

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 1 above)
Comment (vovkandrew) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 2 above)
Comment (vovkandrew) [reply src/main/java/core/basesyntax/FootballMatch.java]: Done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**